### ISSUE TITLE
unable to register an admin account using metalims_empty.sql

### ISSUE THREAD

**Issue by cedarice** [cheinle/metaLIMS#15]
I import metalims_empty.sql to MYSQL following the installation guild. Then, in the Chrome browser, I can open the login page successfully, but when I registered an admin account, it said: Registration submitted. Admin will email you when registration is complete. Please help to resolve this problem, thanks.

**Comment by cheinle** [comment]
Hi Cedarice,

Apologies for the delay in response. Please activate your admin user after your registration by logging in with the default admin user and the following the steps under 'Activate New Admin User' in the Manual for Admin .

https://github.com/cheinle/MetaLIMS/wiki/MetaLIMS-v2.0-Manual-For-Admin#creating-admin-user

Let me know if the problem persists.

Kind regards,

Cassie

**Comment by gador** [comment]
Hi,

 I'm  having a similar issue, although I cannot login as the default admin user. It doesn't even through an error it just keeps returning to the login page. Any ideas why?

Regards,

Florian

**Comment by cheinle** [comment]
Hi Gador,

Thanks for your message.  The first thing that comes to my head is that MetaLIMS has a bit of a quirk on the login page where sometimes you have to actually click the login button instead of using the return/enter key to sign in.  Is it possible that you have tried this?

To better help trouble shoot, could you share with me how you installed MetaLIMS? Did you set up your LAMP stack using the MetaLIMS bash scripts or did you already have a pre-existing set-up? 

It is totally possible that something in the set-up went awry. >.<  One possibility could be that the connection string created in the database connection file (connection.php). Another could be that the mysql db did not get set up with the right setting or permissions .

To test theory 1: 
Please open the connection.php file and check that it looks something like this - 
(if you used the metalims.sh bash script, it should be located one level up in from your webroot directory  in /var/www/metalims_config/ . You can use the command `less /var/www/metalims_config/connection.php` to view the file )

```
$dbc =@mysqli_connect('localhost','metalims','[your password]','metalims_empty') or die ('Could not connect to the 
database because:'.mysqli_connect_error()); 

mysqli_query($dbc, "SET SESSION sql_mode = ''");
```


To test theory 2:

First, ssh into your instance. 

Second, login to your mysql. You can do this by typing `mysql --host=localhost --user=metalims --password=[your password] metalims_empty`

Next, type 
`SELECT * FROM users;`

You should see some output. Make sure it looks something like this (note: sensitive column info has been removed)- 
![select_users](https://user-images.githubusercontent.com/10081573/28496884-28e56dfe-6faa-11e7-8f56-4db85d6fa154.PNG)

If this is there, then check if the permissions for the database are correct. Do this by typing:

`SHOW GRANTS FOR 'metalims'@'localhost';`

The output should look like - 
![show_grants](https://user-images.githubusercontent.com/10081573/28496891-972973a0-6faa-11e7-9525-be87b8edd568.PNG)

Let me know how it goes. I am happy to help further.

Thanks!

Cassie

**Comment by gador** [comment]
Hi, 

thanks for the extensive answer!
Yes, I used both of your scripts on a Debian 8 system. Both worked fine and didn't show any errors. Connection to the MySql DB works. 
I installed phpMyAdmin and checked the table's existence and privileges, both are fine.

Creating a third user (through your webinterface, not phpMyAdmin) works, so it shows up in the user's table. Login is still not possible, even after setting the "visible" status to 1 through phpMyAdmin.

I restarted apache2 and mysqld several times, but it still didn't work. Curiously. after running php --version it worked. ( I have no clue why..).

So thank you very much for the quick response and the help, it works now :)

Florian
